cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/CloudesireClient.java

```diff
@@ -310,11 +310,24 @@ public ResellerPricingApi getResellerPricingApi()
         return getApi( ResellerPricingApi.class );
     }
 
+    public ProceedsReportApi getProceedsReportApi()
+    {
+        return getApi( ProceedsReportApi.class );
+    }
+
+    /**
+     * @deprecated by {@link #getProceedsReportApi()}
+     */
+    @Deprecated
     public ResellingReportApi getResellingReportApi()
     {
         return getApi( ResellingReportApi.class );
     }
 
+    /**
+     * @deprecated by {@link #getProceedsReportApi()}
+     */
+    @Deprecated
     public VendorReportApi getVendorReportApi()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)